### DISCUSSION TITLE
Upload image issue when there are many images

### DISCUSSION THREAD

**Discussion by micropalla** [joomla/joomla-cms#28776] [2. Cold Features]
### Steps to reproduce the issue
Uploading an image in a directory with many images inside.

### Expected result
The image is uploaded.

### Actual result
The image is uploaded in the specified directory only if all images inside that directory have been loaded as thumbs. Otherwise Joomla puts the image in the root. This is bad if you don't have a good connection since you have to wait too much all the images to be loaded.

### System information (as much as possible)
All versions of Joomla have this issue.

### Additional comments
It would be also nice a system to set the attributes of the image once you have uploeded it. Nowadays when you upload an image you have then to find it inside the directory to put it inside the text. There should be a flag like "Upload the image and put it directly in the text".

**Comment by ghost** [comment]
Need label `New Feature` ("Additional comments")
